web_sheet_full_width/README.rst

```diff
@@ -0,0 +1,88 @@
+====================
+Web Sheet Full Width
+====================
+
+.. !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
+   !! This file is generated by oca-gen-addon-readme !!
+   !! changes will be overwritten.                   !!
+   !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
+
+.. |badge1| image:: https://img.shields.io/badge/maturity-Beta-yellow.png
+    :target: https://odoo-community.org/page/development-status
+    :alt: Beta
+.. |badge2| image:: https://img.shields.io/badge/licence-AGPL--3-blue.png
+    :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+    :alt: License: AGPL-3
+.. |badge3| image:: https://img.shields.io/badge/github-OCA%2Fweb-lightgray.png?logo=github
+    :target: https://github.com/OCA/web/tree/15.0/web_sheet_full_width
```

Comment:
I don't know when and how the 15.0 mentions will be replaced by 16.0, maybe when this is merged?

Reply:
I don't know either. I'm going to check it in the case I missed to change it manually

Reply:
The README.rst its generated again when is merged by oca-bot so that mentions to 15.0 will be replaced then.